modules/test/playwright/helpers/SearchExperiencesApiHelper.ts

```diff
@@ -55,31 +40,35 @@ export class SearchExperiencesApiHelper {
 	/**
 	 * It allows creating an SXPBlueprint.
 	 *
+	 * @param title the title of the blueprint
+	 * @param description the description of the blueprint
+	 * @param defaultLanguageId the language id to assign for title/description
 	 * @param configuration Technical configurations that include sort,
 	 * highlight, aggregation, and query configurations
-	 * @param defaultLanguageId the language id to assign for title/description
-	 * @param description the description of the blueprint
 	 * @param elementInstances the elements to be included in the blueprint
-	 * @param title the title of the blueprint
 	 */
 
-	async createSXPBlueprint({
-		configuration = DEFAULT_SXP_BLUEPRINT_CONFIGURATION,
-		defaultLanguageId = 'en_US',
-		description = '',
-		elementInstances = [],
-		title = `Blueprint${getRandomInt()}`,
-	}: createSXPBlueprintProps = {}): Promise<{id: number}> {
+	async createSXPBlueprint(
+		title?: string,
+		description?: string,
+		defaultLanguageId?: string,
+		configuration?: Object,
+		elementInstances?: Object[]
+	): Promise<{id: number}> {
```

Comment:
We should still keep this an object so it's easier to use. Especially since these are all properties on a blueprint, it'll be easier passing in an object rather than making sure values are passed in the correct order. Also it becomes more annoying if you wanted to not pass in certain parameters like `description` you'd have to do something like `createSXPBlueprint('title', null, 'en_US')`

So it'd be something like:
```javascript
async createSXPBlueprint({
	description,
	title,
}: {
	description?: string;
	title?: string;
}) {
	// ...
}
```

Reply:
Oh my bad ;_; 

I'll redo the previous commit because it used to be an object and there's already a lot of commits on this PR

Reply:
Added! Let me know if that's better... still getting used to typescript 🥲